### ISSUE TITLE
I think I'm owed an explanation lol

### ISSUE THREAD

**Issue by WinterrrWarlock** [Oghma-Infinium/Fahluaan#8]
Hey there, first off I'd like to say that Fahluaan is an incredible modlist that is thoughtfully put together. That being said, I've got a bone to pick.

I'd like you to take a gander at the front page. I'm pretty sure it very CLEARLY says "hey, you gotta be on 1170" 

I lost 465 gigs of mods the other day, and it's why I decided to try Fahluaan. Fuck it, did a clean install of Skyrim, downloaded Fahluaan, removed the mods I never use, spent days setting up mine, and started rocking. First thing I noticed was I couldn't issue orders to followers, I'd tell them to do shit, and my character wouldn't be able to move. Alright, every modlist has a quirk, and I don't order followers very much, I can deal.

Then, i started getting realllllly weird DLL errors. Surely has to be the mod problem right? So I went crawling through every subreddit and post I could find trying to find a fix. 

Today, I had an HDT SMP crash. Now, because Fahluaan comes preinstalled with FSMP I decided, no biggie, I'll reinstall and make sure I select the best settings for my system. I reinstall, the fomod opens up, and.... 1.6.640? That's not right. Hmm, maybe the installed version of  FSMP is old. (it was) So off I went to go install the new one. Downloaded it, fomod opens up... 1.6.640. Ok, now i'm confused. I go check my base skyrim install, 1.6.1170... I open the fucking game. 1.6.640

To say I'm pissed off is an understatement. I'm trying to be nice and respectful, but if you're gonna post on the main page that 1170 is a requirement for your modlist, and NOT say "oh by the way Fahluaan comes installed with 1.6.640 skse and the subsequent stock game" then brother I've got some fucking choice words for you. EVERY. SINGLE. DLL. that I have wasted time trying to fix is because there is ZERO documentation about you just fucking picking and choosing SKSE versions. I have spent days trying to figure out what the hell is going on, and you can't be bothered to say "Fahluaan needs 1170 to work, but the actual game in the Stock Game folder, as well as the SKSE version are 1.6.640. Please make sure that any mods you download are for the correct SKSE version." Absolutely fucking kindergarten level shit man. You spent  all this time crafting a GOD TIER modlist that looks nice and plays nice but you can't fucking tell people who download it about the SKSE version? 

I downloaded Fahluaan Dec 8th. 10 days ago. I stared at that fucking main page for hours, reading it. I've been modding Skyrim for years, hell, I've authored mods with 100k+ downloads. Losing my install was a gut punch, but this? This is fucking worse because now I've got to fix YOUR modlist. 

TLDR: Take some damn accountability. Update that fucking mainpage, or better yet, update the modlist to the fucking version that you force us users to install in the first place.

Rant over, Warlock out.

**Comment by Styyx1** [comment]
oh that sounds like someone who has absolutely no clue what they're doing should really not mess with modifying a pre-made modlist and just leave things as they are.

**Comment by aljoxo** [comment]
> This is fucking worse because now I've got to fix YOUR modlist.

Sorry but uh the issue here seems to be that you broke the modlist by heavily modifying it? Like what exactly are you fixing? The modlist works fine in the default configuration, and if you expect me to accommodate usage outside of that then I think you're a bit of an entitled prick.

> I'm pretty sure it very CLEARLY says "hey, you gotta be on 1170"

The readme mentions .1170 because it requires the hashes from the newest steam version of Skyrim to successfully install (which is currently .1170). 

> you can't fucking tell people who download it about the SKSE version?

You failing to do basic due diligence and actually check what game version the list runs in stock game is not my fault. For the average user (doesn't modify the list), there is no reason to have it written out somewhere what the game version of the stock game is. 

> Take some damn accountability. 

Not sure why I need to be held accountable for the modlist breaking due to heavy modifications? We can't and don't support modifications.

> Rant over, Warlock out.

Frankly this entire post comes off as extremely entitled and childish on your part.

**Comment by iAmMe27** [comment]
⠀⠀⠀⠀⠀⠀⠀⠀⠀⢀⠤⠒⠉⠉⠉⠉⠉⠉⠉⠉⠉⠀⠂⢄⡀⠀⠀⠀⠀⠀
⠀⠀⠀⠀⠀⠀⠀⡠⠊⠁⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠈⠒⢀⠀⠀⠀
⠀⠀⠀⠀⠀⡠⠊⠠⠒⠉⠉⠉⠉⠐⢄⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠠⡀⠀
⠀⠀⠀⠀⡰⠁⡔⠁⠀⠀⠀⠀⠀⠀⠀⢃⠀⠀⠀⠀⢀⠄⠒⠒⠒⠒⢄⠀⠘⠄
⠀⠀⠀⠠⠀⠀⡇⠀⠀⠀⠀⠀⢀⣀⡀⢸⠀⠀⠀⠔⠁⠀⠀⠀⠀⠀⠀⠡⠀⢰
⠀⠀⠀⡇⠀⠀⠡⡀⠀⠀⠀⠀⠸⠿⠏⡜⠀⠀⢸⠀⠀⠀⠀⠀⢀⣀⠀⠀⠃⢸
⠀⠀⠀⡇⠀⠀⠀⠑⠄⢀⡀⢀⡀⠠⠊⠀⠀⠀⠘⡀⠀⠀⠀⠀⢿⣿⠇⢀⠇⢸
⠀⠀⠀⡇⠀⠀⠀⠀⠀⢀⠤⣀⣀⡀⠀⠀⣠⣤⡄⠐⢄⠀⠀⠀⠀⠀⡠⠊⠀⢸
⠀⠀⠀⡇⠀⠀⠀⠀⢠⠃⢀⠤⢀⡈⠉⣾⡟⠁⠀⠀⠀⠈⠁⠀⠈⠁⠀⠀⠀⢸
⠀⠀⠀⣱⡤⣄⣀⢠⠃⢀⠇⠀⠀⠈⢹⣿⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⣸
⠀⠀⢠⠁⠀⠀⠀⠉⠁⠚⠢⢄⠀⠀⠘⣿⣆⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⢰⠃
⠀⢀⠃⠀⠀⠀⠀⡖⠢⢤⣀⠀⠳⡀⠀⠈⠛⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⡠⠃⠀
⠀⡌⠀⠀⠀⣠⠤⣈⡢⡀⠀⠉⠛⡇⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⡠⠊⠀⠀⠀
⡴⠀⠀⠀⠀⠳⣄⠀⠈⠁⠀⠀⠀⠐⢄⠀⠀⠀⠀⠀⢀⣀⠠⠒⠉⠀⠀⠀⠀⠀
⡇⠀⠀⠀⠀⡤⠤⢍⣲⡄⠀⠀⠀⠀⠈⢹⠉⠉⠉⠉⠁⠀⠀⠀⠀⠀⠀⠀⠀⠀
⢧⣄⣀⠀⠀⠈⠲⠤⣀⠀⠀⠀⠀⠀⠀⠀⡇⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
⠀⠹⢿⣿⣷⣶⣤⣤⡀⢳⡀⠀⠀⠀⠀⠀⡇⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
⠀⠀⠀⠀⠉⠉⠛⠛⠿⠦⣷⢄⣀⣀⡀⠜⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀

**Comment by Mieekaserra** [comment]
> I'd like you to take a gander at the front page. I'm pretty sure it very CLEARLY says "hey, you gotta be on 1170" 

Yes. thats a standard requirement of any featured modlist, you gotta be 1170 to install.

None of the lists *stay on that version*

> removed the mods I never use, spent days setting up mine, and started rocking. 

This was your first mistake. 

> To say I'm pissed off is an understatement. 

 you edited the modlist without doing due diligence to check what version you are modding. I'd say thats your own Fault.


> I'm trying to be nice and respectful, but if you're gonna post on the main page that 1170 is a requirement for your modlist, and NOT say "oh by the way Fahluaan comes installed with 1.6.640 skse and the subsequent stock game" then brother I've got some fucking choice words for you.  ... I have wasted time trying to fix is because there is ZERO documentation about you just fucking picking and choosing SKSE versions.

Problem. In. Chair. Not. In. Modlist.

That would be because 99% of wabbajack lists don't support modifications and are done at your own risk.

> I have spent days trying to figure out what the hell is going on, and you can't be bothered to say "Fahluaan needs 1170 to work, but the actual game in the Stock Game folder, as well as the SKSE version are 1.6.640. 

They arent obligated to.


> I downloaded Fahluaan Dec 8th. 10 days ago. I stared at that fucking main page for hours, reading it. I've been modding Skyrim for years, hell, I've authored mods with 100k+ downloads. Losing my install was a gut punch, but this? This is fucking worse because now I've got to fix YOUR modlist.

WHICH YOU BROKE IN THE FIRST PLACE BY NOT. CHECKING.

TLDR: Check yourself  before bitching and doing unsupported stupid shit.

**Comment by WinterrrWarlock** [comment]
Let me clarify the things I added. MCO-ADXP, DMCO, Nether's Follower Framework, Simple Offense Suppression, Elden Power Attack. Things I removed, the INI setting for Wheeler as I don't use it. TK Dodge, because I prefer DMCO. I added basic Adult framework, SL, Acheron, Yamate, Pandora, animations. That's it. I haven't heavily modded anything, in fact I made sure that the load order of the install stayed exactly the same because I know the kind of work that goes into the custom rulesets that mod authors create. The new esps were sorted by hand to ensure no conflicts, they were put at the end of the list right before the DYNDOLOD plugins. If a list requires 1170, then it's a pretty fair bet that it's an 1170 SKSE. 

You are right on a couple of things, my frustration came off as childish and immature, and for that I truly do apologize. I let my level of frustration cloud both by judgment and my reasoning, and my choice of communication is both less than ideal and frankly piss poor. However, I do hope that you can understand at least part of my frustration. A simple edit to the main page at the very least would be nice. The amazing thing about Wabbajack lists is that you don't have to manually go and find every tiny little patch buried on Nexus because someone has spent hours doing that. It's incredible. The bad thing about Wabbajack lists is occasionally there are things that are frankly useless to the end user (ie. Wheeler) for me. BEES is an incredible tool for backwards compatibility, but obviously a DLL file for the actual version of the SKSE you're running is much better. I've been modding Skyrim for a decade plus at this point, and the quality of mods have skyrocketed in that time to say the least.

My actions were entitled, and that was wrong. I take full ownership of sounding like a bitch. My original point though I believe has some merit, and I do think that some information for those wanting to mod things for themselves could go a long way. Some people might like different NPC replacers, some might prefer a different combat setup. All those things are relatively easy to fix. It's the SKSE plugins that are always the problem. Modding an existing wabbajack comes with risks, absolutely. Some of those risks could be mitigated by a simple "Fahluaan uses 1.6.640 for it's Stock Game" 

Once again, apologies for my actions and my words. I ain't perfect, and shouldn't have let my temperament be clouded to the degree it was.

**Comment by EzioTheDeadPoet** [comment]
Mentioning any other version than what people need installed in steam for installing a list does more harm than good to the average user that doesn't know anything about modding and uses a Wabbajack list for that exact reason.
It is expected of people that want to modify lists to be skilled enough to properly identify what WJ installed and how to mod it on their own.

People that want to modify setups are not the target audience except for a select number of lists that advertise as such.
When people want to modify stuff they are doing so at their own risk and on their own.

Supporting stuff like this is beyond the scope of a free volunteer maintained offering.

**Comment by WinterrrWarlock** [comment]
Perhaps, but as was stated earlier the average person using the Wabbajack list wouldn't be modding it. It doesn't excuse my actions or words, that ownership is entirely mine. However, stating that it's beyond the scope to just add a small note seems a little strange. You're right, at any point I could have gone into the stock game EXE and checked the version number, or seen the tiny SKSE version in the system menu. I guess the point that I'm trying to make is that I went looking through the readme, the changelog, the main page, all the documentation before trying everything else. If there had been any sort of notice, I would have quickly realized the discrepancy, instead of discovering it by almost happenstance. Once again my apologies for the actions of earlier, I was correctly criticized. I still believe that my point has some level of merit when it comes to documentation :)

**Comment by Megumints** [comment]
> However, stating that it's beyond the scope to just add a small note seems a little strange.

It's not the list author's responsibility for this. The list author ensures that the list, in its supported configuration, is installable and playable. Hence the note stating if you're not on the latest version then the list will not install correctly. Your ask goes beyond this scope and has no benefit for an end user who will just play the list as intended.  If you're going to modify the list you must perform your due diligence yourself ensuring that your additions are compatible.

> I still believe that my point has some level of merit when it comes to documentation :)

The intent and purpose of the documentation is to guide the user to successfully install and play the list in its supported configuration.

**Comment by WinterrrWarlock** [comment]
Thank you for the feedback that I received everyone. Once again, apologize for being an ass! This can be marked as closed. I understand the position a lot of you took, this wabbajack is designed for an "install and go" approach, and for those that works for that's awesome. Fahluaan was truly gorgeous, and gave me a few ideas for mods that I really liked and will probably be adding to a new modlist. None of you deserved any of my frustration, and for that I apologize. I definitely didn't leave a good first impression, and that's a regret for sure. Gonna give NVGO a try, because it uses 1170 in the stock game and actively encourages building on the list. :) Fahluaan is beautiful, but not for me. I wish you all well, and hope that SSE never CTD's on you again :)
